### PR TITLE
Dressup simulator

### PR DESCRIPTION
**Pull request by Lazejun**
## What type of PR is this.
1.  Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added more Uniforms/Headgear to the new Arrays in Reb/Civ Templates,
Changed EquipRebell to use the RebellHeadgear array instead of UnlockedHeadGear,
------> This is to avoid Possible PID issues and to give Rebell AI a Faction fitting look.

Renamed 2 Arrays:
AllCivilianGlasses and AllCivilianHeadgear are now called AllCosmeticHeadgear and AllCosmeticGlasses.
These are the Autodetection Arrays given to the Arsenal, the list will be excessive as before, but this is so player can do their normal Barbie Dressup.
-----> Basicly returning it to how it was before GeneralHasVN

Added 2 False NVGs to the Unknown Category to avoid them spawning.

Split 3CB CIV into Arid and Temperate:
Arid will be Takistani Civs, with their Vehicles, Uniforms and Hats
Temperate are Chernarussian Civs, with their Vehicles, Uniforms and Hats.

Changes to fn_equipmentIsValidForCurrentModset.sqf:
Apparently those were broken with General VN,
As it looked the Previous behavior would have allowed Vets below Armor 5 and Hats with 0 Armor,
Somehow that didn't realy work, so i changed it to allow Vests below Armor 12.1 and Hats below 0.1.
8 Armor is on Tac Vets and 12 Armor is on Police Vests from Vanilla which RHS uses too.

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2.  Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

File: A3-Antistasi/Templates/NewTemplates/RHS/RHS_Reb_NAPA_Temperate.sqf [hunk 1]
```diff
@@ -49,14 +49,49 @@
...
+	"rhsgref_uniform_TLA_1",
+	"rhsgref_uniform_TLA_2",
+	"rhs_insurgent_uniform_1",
+	"rhs_insurgent_uniform_2",
+	"rhs_insurgent_uniform_3",
+	"rhs_insurgent_uniform_4",
+	"rhs_insurgent_uniform_5"
+]] call _fnc_saveToTemplate;
+
+	["headgear", [
```
Comment (jaj22) [inline A3-Antistasi/Templates/NewTemplates/RHS/RHS_Reb_NAPA_Temperate.sqf:83]: Gained a tab here, and in a few similar files.

File: A3-Antistasi/functions/Ammunition/fn_equipmentIsValidForCurrentModset.sqf [hunk 2]
```diff
@@ -29,10 +29,10 @@ if (_itemIsVanilla && {A3A_hasRHS}) exitWith {
 		case "Equipment": {
 			switch (_categories select 1) do {
 				case "Headgear": {
-					if (getNumber (_configClass >> "ItemInfo" >> "HitpointsProtectionInfo" >> "Head" >> "armor") > 0) then { false };
+					if (getNumber (_configClass >> "ItemInfo" >> "HitpointsProtectionInfo" >> "Head" >> "armor") < 0.1) then { true };
 				};
 				case "Vest": {
-					if (getNumber (_configClass >> "ItemInfo" >> "HitpointsProtectionInfo" >> "Chest" >> "armor") > 5) then { false };
+					if (getNumber (_configClass >> "ItemInfo" >> "HitpointsProtectionInfo" >> "Chest" >> "armor") < 12.1) then { true };
```
Comment (jaj22) [inline A3-Antistasi/functions/Ammunition/fn_equipmentIsValidForCurrentModset.sqf:35]: This seems... generous. But ok. Should makes squad AIs a bit more useful in the early game.

File: A3-Antistasi/Templates/NewTemplates/3CB/3CB_Civ_Arid.sqf [hunk 3]
```diff
@@ -3,35 +3,30 @@
...
+    "UK3CB_TKC_C_U_02",
+    "UK3CB_TKC_C_U_02_B",
+    "UK3CB_TKC_C_U_02_C",
+    "UK3CB_TKC_C_U_02_D",
+    "UK3CB_TKC_C_U_02_E"
 ]] call _fnc_saveToTemplate;
 
-["headgear", [""]] call _fnc_saveToTemplate;
+["headgear", [
+	"UK3CB_TKC_H_Turban_01_1",
```
Comment (jaj22) [inline A3-Antistasi/Templates/NewTemplates/3CB/3CB_Civ_Arid.sqf:19]: Uniforms are 4-space, headgear are tabs. The new templates seem to be mostly tab-spaced but I don't really care as long as similar stuff is consistent.

Applies generally to the PR. In some cases it's switching mid-array, which is pretty obvious in the git diff.

**Review by jaj22 — APPROVED** [review]
Just formatting issues.

File: A3-Antistasi/Templates/NewTemplates/3CB/3CB_Civ_Arid.sqf (hunk 3 above)
Comment (Lazejun) [reply A3-Antistasi/Templates/NewTemplates/3CB/3CB_Civ_Arid.sqf:19]: 🙄 will go over it later

File: A3-Antistasi/functions/Ammunition/fn_equipmentIsValidForCurrentModset.sqf (hunk 2 above)
Comment (Lazejun) [reply A3-Antistasi/functions/Ammunition/fn_equipmentIsValidForCurrentModset.sqf:35]: That just allows the Items to apear in Loot, not realy related to Ai Equipment or Starting Gear

**Review by Bob-Murphy — APPROVED** [review]

**Pull request merged**